docs/components/modeler/desktop-modeler/element-templates/defining-templates.md

```diff
@@ -73,6 +77,40 @@ For example, given the following `$schema` definition, the application takes `0.
 
 The JSON schema versioning is backward-compatible, meaning that all versions including or below the current one are supported.
 
+## Execution platform compatibility
```

Comment:
This is confusing because the term "execution platform" is reserved for runtimes: Camunda 7, Camunda 8 etc.
Perhaps "Template compatibility" conveys the message better?